lib/src/model/ChallengeError.g.dart

```diff
@@ -10,7 +10,7 @@ ChallengeError _$ChallengeErrorFromJson(Map<String, dynamic> json) =>
     ChallengeError(
       detail: json['detail'] as String?,
       type: json['type'] as String?,
-      status: json['status'] as String?,
```

Comment:
The code is generated so that will be a problem.

For the moment I've  changed the status to int and that has solved the issue but of course it might break other things.

Reply:
The status is int? so optional is fine. From what I can tell from the RFC it looks like the status is always an int so I think the change of pushed should be ok.

Reply:
Are you sure that the status is optional ? I took a quick look on the rfc and can not only find "status" fields that are marked as required.

Reply:
There are errors in the RFC without a status.